### PR TITLE
Enhanced glances widget

### PR DESCRIPTION
**Pull request by GAntoine**
## Proposed change

Enhance the glances widget to include the ability to display disk(s) and mirror the style and structure of the resource widget.

Closes #672 

235331114-69b2dc0b-4d2e-4ec4-91e3-db5ac66ce621

## Type of change

- [ ] New service widget
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Other (please explain)

## Checklist:

- [x] If adding a service widget or a change that requires it, I have added a corresponding PR to the [documentation](https://github.com/benphelps/homepage-docs) here: https://github.com/benphelps/homepage-docs/pull/104
- [x] If adding a new widget I have reviewed the [guidelines](https://gethomepage.dev/en/more/development/#service-widget-guidelines).
- [x] If applicable, I have checked that all tests pass with e.g. `pnpm lint`. ✔ No ESLint warnings or errors
- [x] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.

### CODE REVIEW

**Comment by shamoon** [comment]
Thanks.

The reason I did not implement this yet is because it increases the API calls quite significantly, these widgets poll the API every 1.5 seconds... Theres obviously no avoiding this but I did update things to actually not hit the API if not needed (had been meaning to make that change but now more relevant). Also one small issue with 

Otherwise, looks good

File: src/pages/api/widgets/glances.js [hunk 1]
```diff
@@ -40,25 +40,31 @@ async function retrieveFromGlancesAPI(privateWidgetOptions, endpoint) {
 }
 
 export default async function handler(req, res) {
-  const { index } = req.query;
+  const { index, cputemp: includeCpuTemp, uptime: includeUptime, disk: includeDisks } = req.query;
```
Comment (GAntoine) [inline src/pages/api/widgets/glances.js:43]: Ahhh smart! I tried doing this earlier but struggled to find where I could access the settings 👍🏼

**Comment by luweijun1992** [comment]
v0.6.21 When will it be released?

**Comment by mgrimace** [comment]
Can it be possible to 1) optionally hide cpu and mem (i.e., to just show storage), and/or 2) re-order items based on the order of the items listed in the widgets.yaml. Thank you!

**Comment by GAntoine** [comment]
1) you can set the fields shown as per the documentation 

2) reordering fields is not supported in any widget, and would need a larger body of work to make it possible

**Pull request merged**